node/cli/src/service.rs

```diff
@@ -83,8 +82,12 @@ construct_service_factory! {
 				let (block_import, link_half) = service.config.custom.grandpa_import_setup.take()
 					.expect("Link Half and Block Import are present for Full Services or setup failed before. qed");
 
-				if let Some(aura_key) = service.authority_key::<AuraPair>() {
-					info!("Using aura key {}", aura_key.public());
+				if service.config.roles == Roles::AUTHORITY {
```

Comment:
This is not correct, you might want to run the grandpa voter as a full node (in fact in master we just added a flag for this).